src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -1,22 +1,38 @@
 package mate.academy.bookstore.model;
 
+import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Data
 @Entity
+@Table(name = "books")
+@NoArgsConstructor
```

Comment:
You don't have any constructors with parameters here, so default constructor will be created by default